xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -81,12 +81,15 @@ struct ValidatedFormField<Output: Equatable, Caption: View>: View {
                     invalidBadge
                 }
             }
-            caption
-                .foregroundColor(
-                    field.shouldPresentAsInvalid ? Color.red : Color.secondary
-                )
-                .animation(.default, value: field.shouldPresentAsInvalid)
-                .font(.caption)
+            if let caption = caption {
+                Text(String(localized: caption))
```

Comment:
FYI, Text automatically localizes static strings.

Reply:
If we pass the string in dynamically, we want to use `String(localize:)` and `Text(verbatim:)`